src/main/java/com/bugsnag/android/NativeInterface.java

```diff
@@ -0,0 +1,184 @@
+package com.bugsnag.android;
+
+import java.sql.BatchUpdateException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Used as the entry point for native code to allow proguard to obfuscate other areas if needed
+ */
+class NativeInterface {
+
+    public static String getContext() {
+        return Bugsnag.getClient().getContext();
```

Comment:
Ah - guessing that we are using static `Bugsnag` everywhere to make this whole interface possible. Any workarounds you can think of that would avoid depending on the static class?